prov/gni/src/gnix_ep.c

```diff
@@ -1864,6 +1929,22 @@ static ssize_t gnix_ep_tx_size_left(struct fid_ep *ep)
 	return 64;
 }
 
+__attribute__((unused))
```

Comment:
I can't tell from looking at the Jenkins build, either gnix_ep.c is not being compiled or I'm missing something.  However, It looks like at least clang 3.6 and 3.9 support this.